pgrx-tests/src/tests/memcxt_tests.rs

```diff
@@ -97,4 +98,127 @@ mod tests {
         drop(ctx);
         assert_eq!(unsafe { pg_sys::CurrentMemoryContext }, ctx_parent);
     }
+
+    #[cfg(feature = "nightly")]
+    #[pg_test]
+    #[no_mangle]
```

Comment:
hm. why did you add `#[no_mangle]` to these?

Reply:
Probably doesn't help, I just wanted to make sure this doesn't optimize the behavior we actually want to test out of existence because the test is too structurally simple or w/e. And so it is probably not necessary.